LICENSE

```diff
@@ -22,3 +50,29 @@ ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
 
 For more information, please refer to <http://unlicense.org/>
+
+MIT license
+-----------
+The MIT License (MIT)
+
+C̶o̶p̶y̶r̶i̶g̶h̶t̶ ̶(̶c̶)̶ 2013-2017 Luke Sampson (https://github.com/lukesampson)
+C̶o̶p̶y̶r̶i̶g̶h̶t̶ ̶(̶c̶)̶ 2013-present Scoop contributors (https://github.com/ScoopInstaller/Scoop/graphs/contributors)
+Copyright (c) 2022-present Scoop contributors
```

Comment:
This is ready to merge, but I'm not sure why there is strikeout, and a repeating line. I would suggest it simply read:
```
Copyright (c) 2013-2017 Luke Sampson (https://github.com/lukesampson)
Copyright (c) 2013-2022 Scoop contributors (https://github.com/ScoopInstaller/Scoop/graphs/contributors)
```
Or drop the ending year:
```
Copyright (c) 2013 Luke Sampson (https://github.com/lukesampson)
Copyright (c) 2013 Scoop contributors (https://github.com/ScoopInstaller/Scoop/graphs/contributors)
```
but including the initial year is advisable per [here](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice) and [here](https://www.copyrightlaws.com/copyright-symbol-notice-year/#h-q-a-copyright-notice-year).

Reply:
I prefer former ones that have ending year, and the `2022` should just be `present`, e.g., `2013-present`. (as [Babel's](https://github.com/babel/babel/blob/master/LICENSE))

Or just like [.NET's](https://github.com/dotnet/runtime/blob/main/LICENSE.TXT), remove the starting and ending year.

Reply:
> but I'm not sure why there is strikeout, and a repeating line

I've the same question, what's the strikeout mean? @Tyler887